react-stopwatch/my-app/src/StopWatch.tsx

```diff
@@ -0,0 +1,59 @@
+import { useState } from 'react';
+import { FaPlay, FaPause } from 'react-icons/fa';
+import './StopWatch.css';
+
+export function StopWatch() {
+  const [elapsedSecs, setElapsedSecs] = useState(0);
+  const [intervalId, setIntervalId] = useState<NodeJS.Timeout | null>(null);
+
+  const startTimer = () => {
```

Comment:
Nested arrow functions??? Ewwww! ;-)

I'm teasing because I don't like nested arrow functions. I only use arrow functions when I'm passing a function as an argument to another function. For standalone and nested functions, I always use `function` syntax. But that's just because I think it is easier to read that way. You do _not_ have to follow my personal guidelines. You should do what you are most comfortable with. And, yes, a lot of people disagree with me and always use arrow functions. (I happen to think they're somewhat odd, though....... :-)